src/strategies/index.ts

```diff
@@ -0,0 +1,10 @@
+import {FluentStrategy} from './FluentStrategy'
+import {Random, Dedupable, Biased, Cached, Shrinkable} from './FluentStrategyMixins'
+
+export const RandomCachedStrategy = Cached(Dedupable(Random(FluentStrategy)))
+
+export const RandomCachedStrategyWithShrinking = Shrinkable(Cached(Dedupable(Random(FluentStrategy))))
+
+export const BiasedRandomCachedStrategy = Cached(Biased(Dedupable(Random(FluentStrategy))))
+
+export const BiasedRandomCachedStrategyWithShrinking = Shrinkable(Cached(Biased(Dedupable(Random(FluentStrategy)))))
```

Comment:
These names and way of composing strategies is so bad. Please, at least, open an issue to transform this into a fluent interface.

```
fc.scenario().withBias().usingCache().withRandomSampling().
  forall(...
```

Reply:
No need, I can make the changes in this PR.

Reply:
The way of composing strategies is how it is supposed to be when using mixins. I understand your point of view that it may not be the clearest thing ever. However, even providing your suggested interface on the backend the composition would be the same. 

The question here is: do we want users to be able to configure strategies or to select strategies? I was thinking about implementing the latter since the strategy composition of the former involves knowing the logic and behavior behind the mixins.

Reply:
FluentCheck is called that way because it provides a Fluent interface.

And I think developers are grown enough to read some documentation. Just give them decent defaults, and provide some examples.

Reply:
Ok, I will change the approach taken to be more fluent then and to be similar to what you have specified.

Reply:
Already made the required changes.